32_workflow-make-github-repo-browsable.Rmd

```diff
@@ -163,6 +163,16 @@ Examples:
   
 Hopefully we are moving towards a world where you can have "web friendly" and "vector" at the same time, without undue headaches. As of [October 2014](https://github.com/blog/1902-svg-viewing-diffing), GitHub provides enhanced viewing and diffing of SVGs. So don't read this advice as discouraging SVGs. Make them! But consider keeping a PNG around as emergency back up for now.
   
+## Other document formats
+
+You may also have a document you want others to be able to browse and interact with, but it is not in the markdown format. Fortunately, the open-source Pandoc program, written by John MacFarlane, allows you to convert a range of formats into markdown, including the widely used docx format.
+
+When you click the Knit button in Rstudio it is actually Pandoc which performs the final conversion to HTML or word (docx) formats. If you are willing to use the command-line, you can perform the opposite conversion (eg docx to markdown), commonly retaining features such as headings, tables, equations and even figures.
+
+As some boilerplate, running in Windows PowerShell `pandoc --extract-media .\media  -f docx .\example.docx -t markdown_github -o example_image.md` converts a word document called example.docx to markdown, and extracts the images into a directory which corresponds to a filepath in the newly created example.md document. A full list of supported formats and example code for conversions are available at https://pandoc.org/.
+
+For simpler conversions between the different markdown flavours supported by Pandoc (mmd, php_extra and strict), you can convert these to Github using the Rstudio gui by simply renaming their extension to .Rmd.
```

Comment:
I think this is incomplete. You still need to add YAML front matter, then render to md. Can you add a sentence or two about that and link here for more detail:

http://happygitwithr.com/rmd-test-drive.html#output-format

Otherwise, I'll leave this open until I can merge and immediately do this myself.